windows/autobuild/autobuild_windowsinstaller_2_build.ps1

```diff
@@ -0,0 +1,21 @@
+# Sets up the environment for autobuild on Windows
+
+# Get the source path via parameter
+param (
+	[string] $jamulus_project_path = $Env:jamulus_project_path
+)
+# Sanity check of parameters
+if(($jamulus_project_path -eq $null) -or ($jamulus_project_path -eq "")){
+	throw "expecting ""jamulus_project_path"" as parameter or ENV"
```

Comment:
Not sure about the double quotes here? Don't you need + or . to add the content of the jamulus_project_path variable here? Not a PS expert though.

Reply:
this errormessage shall contain the variable-name nof whats missing, so no . / + here

Reply:
Ok. So are two `"` the right syntax here?

Reply:
yes.
results in this message:
![grafik](https://user-images.githubusercontent.com/6545169/107886169-84946b80-6efe-11eb-9dbd-5f0344169b09.png)